src/accelerate/accelerator.py

```diff
@@ -143,6 +143,10 @@ class Accelerator:
             The number of steps that should pass before gradients are accumulated. A number > 1 should be combined with
             `Accelerator.accumulate`. If not passed, will default to the value in the environment variable
             `ACCELERATE_GRADIENT_ACCUMULATION_STEPS`.
+        adjust_scheduler_to_accumulation (`bool`, *optional*, defaults to `False`):
+            Whether or not the scheduler should be adjusted to the number of gradient accumulation steps. If `True`,
+            the scheduler's length will be reduced by `gradient_accumulation_steps`. If the scheduler already accounts
+            for the number of gradient accumulation steps, this should be set to `False`.
```

Comment:
We're starting to have too many arguments to `Accelerator`. This one should be documented waaaay below as it's not as important, and we should think of grouping it with the gradient_accumulation_steps in a `GradientAccumulationPlugin` maybe.